Makefile

```diff
@@ -1,13 +1,13 @@
 INTEGRATION     := redis
 BINARY_NAME      = nri-$(INTEGRATION)
 SRC_DIR          = ./src/
-GO_PKGS         := $(shell go list ./... | grep -v "/vendor/")
+#GO_PKGS         := $(shell go list ./... | grep -v "/vendor/")
 VALIDATE_DEPS    = golang.org/x/lint/golint
 TEST_DEPS        = github.com/axw/gocov/gocov github.com/AlekSi/gocov-xml
 INTEGRATIONS_DIR = /var/db/newrelic-infra/newrelic-integrations/
 CONFIG_DIR       = /etc/newrelic-infra/integrations.d
 GO_FILES        := ./src/
-WORKDIR         := $(shell pwd)
+#WORKDIR         := $(shell pwd)
```

Comment:
why we want to have this commented code?

Reply:
Nice catch, will remove it